src/screens/HyperSwitch/Customers/Customers.res

```diff
@@ -0,0 +1,50 @@
+@react.component
+let make = () => {
+  open APIUtils
+  open CustomersEntity
+  let fetchDetails = useGetMethod()
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+  let (customersData, setCustomersData) = React.useState(_ => [])
+  let (offset, setOffset) = React.useState(_ => 0)
+
+  let getCustomersList = async () => {
+    try {
+      setScreenState(_ => PageLoaderWrapper.Loading)
+      let customersUrl = getURL(~entityName=CUSTOMERS, ~methodType=Get, ())
+      let response = await fetchDetails(customersUrl)
+      let data = response->LogicUtils.getArrayDataFromJson(itemToObjMapper)
+      setCustomersData(_ => data->Array.map(Js.Nullable.return))
+      setScreenState(_ => PageLoaderWrapper.Success)
+    } catch {
+    | Js.Exn.Error(e) =>
+      let err = Js.Exn.message(e)->Belt.Option.getWithDefault("Failed to Fetch!")
+      setScreenState(_ => PageLoaderWrapper.Error(err))
+    }
+  }
+  React.useEffect0(() => {
+    getCustomersList()->ignore
+    None
+  })
+
+  <ErrorBoundary>
```

Comment:
why <ErrorBoundary> is added over here

Reply:
It's just a standard and we should add in every individual components that if some error happened in Customers Module the whole app won't get crashed.

Reply:
Expect api error we don't expect any error in the individual components.And also  <ErrorBoundary> is added at the App level right?

Reply:
Yes it is added in the main entry component. So then we can remove from everywhere. Like Table tabs analytics and every component

Reply:
If its added in the main entry component then we can remove from here right